macosx/HandBrake.xcodeproj/project.pbxproj

```diff
@@ -328,6 +328,7 @@
 		A9F217E61E2F934C00C10C6E /* container-migration.plist in Resources */ = {isa = PBXBuildFile; fileRef = A9F217E51E2F934C00C10C6E /* container-migration.plist */; };
 		A9F472891976B7F30009EC65 /* HBSubtitlesDefaultsController.m in Sources */ = {isa = PBXBuildFile; fileRef = A9F472871976B7F30009EC65 /* HBSubtitlesDefaultsController.m */; };
 		A9F7102619A475EC00F61301 /* HBDockTile.m in Sources */ = {isa = PBXBuildFile; fileRef = A9F7102519A475EC00F61301 /* HBDockTile.m */; };
+		D0924626280B72BA00B79A40 /* Localizable.strings in Resources */ = {isa = PBXBuildFile; fileRef = D0924624280B72BA00B79A40 /* Localizable.strings */; };
```

Comment:
I think this is the usual duplicated Localizable.strings that Xcode adds for some reasons at the top of the project.

Reply:
You are right. I removed it. And I removed the other string that Xcode adds for some reasons.